analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -307,6 +307,14 @@ class PIP : PackageManager() {
         )
     }
 
+    private fun getSourceArtifact(pkgReleases: ArrayNode): RemoteArtifact {
+        val pkgSource = pkgReleases.asSequence().find { it["packagetype"].asText() == "sdist" } ?: return RemoteArtifact.EMPTY
+        val url = pkgSource["url"]?.asText() ?: return RemoteArtifact.EMPTY
+        val hash = pkgSource["md5_digest"]?.asText() ?: return RemoteArtifact.EMPTY
+
+        return RemoteArtifact(url = url, hash = hash, hashAlgorithm = HashAlgorithm.MD5)
```

Comment:
Maybe in this case, as it's quite obvious, consider dropping the named arguments for shortness.